communications/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
-# Create your models here.
+
+class ChatRoom(models.Model):  # Create your models here.
+    id = models.AutoField(primary_key=True)
+    company_id_1 = models.ForeignKey(
+        "companies.CompanyProfile",
+        on_delete=models.SET_NULL,
+        related_name="company1",
+        null=True,
+    )
+    company_id_2 = models.ForeignKey(
+        "companies.CompanyProfile",
+        on_delete=models.SET_NULL,
+        related_name="company2",
+        null=True,
+    )
+    mongo_room_id = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        unique_together = ("company_id_1", "company_id_2")
+
+    def __str__(self):
+        return f"{self.company_id_1} - {self.company_id_2}"
```

Comment:
Right now, (A, B) and (B, A) are treated as two separate ChatRooms. If the logic assumes bidirectional uniqueness (i.e., a chat between A and B is the same as between B and A), you should enforce ordering in your code or override `.save()`:
```
def save(self, *args, **kwargs):
    if self.company_id_1 and self.company_id_2:
        if self.company_id_1.id > self.company_id_2.id:
            self.company_id_1, self.company_id_2 = self.company_id_2, self.company_id_1
    super().save(*args, **kwargs)
```
This way, the pair (1, 2) and (2, 1) will always be saved in the same order.